bazel/grpc_deps.bzl

```diff
@@ -217,11 +217,11 @@ def grpc_deps():
         http_archive(
             name = "zlib",
             build_file = "@com_github_grpc_grpc//third_party:zlib.BUILD",
-            sha256 = "ef47b0fbe646d69a2fc5ba012cb278de8e8946a8e9649f83a807cc05559f0eff",
-            strip_prefix = "zlib-21767c654d31d2dccdde4330529775c6c5fd5389",
+            sha256 = "90f43a9c998740e8a0db24b0af0147033db2aaaa99423129abbd76640757cac9",
```

Comment:
FTR, this is upgrading from https://github.com/madler/zlib/releases/tag/v1.2.12 to https://github.com/madler/zlib/releases/tag/v1.2.13